src/control/server/config/server.go

```diff
@@ -38,8 +38,9 @@ const (
 // See utils/config/daos_server.yml for parameter descriptions.
 type Server struct {
 	// control-specific
-	ControlPort     int                       `yaml:"port"`
-	TransportConfig *security.TransportConfig `yaml:"transport_config"`
+	ControlInterface string                    `yaml:"control_iface"`
```

Comment:
I think this should probably have the `,omitempty` tag, right?

Reply:
Good catch.